plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/completion/SQLCompletionAnalyzer.java

```diff
@@ -938,7 +940,7 @@ private enum InlineState {
     };
 
     @Nullable
-    private Pair<String, String> extractTableName(@Nullable String tableAlias, boolean allowPartialMatch) {
+    private List<Pair<String, String>> extractTableNames(@Nullable String tableAlias, boolean allowPartialMatch) {
```

Comment:
This method is still marked as nullable but its return type is treated as not-null without any checks in several places. I beg you to use IntelliJ IDEA for development, at least it has nullability inspections that would have prevented you from doing this.

But now all its `null` paths can now return empty lists instead.

Reply:
Thank you, I forgot to change nullable annotation. Now It's changed to NotNull.

Actually, IDEA says nothing to me. And I don't quite understand which "null paths" are you talking about, because all usages of this method was fixed.

Reply:
I guess that's because you haven't configured annotation processing yet.

![image](https://user-images.githubusercontent.com/35821147/199491954-d3951762-bffa-4c3f-81a7-cd7d978bd766.png)

Reply:
Yeah, thanks.